README.md

```diff
@@ -43,35 +43,23 @@ Install new Python dependencies with pip
 
 ```make requirements-dev```
 
-Install frontend dependencies with npm and gulp
+Install frontend dependencies with yarn and gulp
 
 ```
-npm install
+yarn
```

Comment:
Following the instructions here doesn't work for me as I don't have yarn installed - the readme probably needs something about how to install yarn in the first place?

Reply:
Oops missed that bit! You can install it with brew on a mac

Reply:
I'll add this link: https://yarnpkg.com/en/docs/install